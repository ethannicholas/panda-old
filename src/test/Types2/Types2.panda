var a:(Int)=>()
var b:(String)