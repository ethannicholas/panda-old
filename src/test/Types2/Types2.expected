error: Types2.panda:1:7: functions must have a return type
error: Types2.panda:2:7: tuple types must have at least two elements
2 errors
